### PR TITLE
Multiple enhancements

### PR DESCRIPTION
**Pull request by dvkch**
Hello everyone,

This PR has for purpose to add the following enhancements:
- Added iOS 9 support
- Fixed picker toolbar wrong positioning on iOS 11
- Added animation to picker appearance/disappearance
- Updated gitignore

Notes:
- `README.md` does not reflect the proper minimum supported iOS version
- `SmartcarAuth` project has multiple times the example app, once as a subproject, once as a target. I'd recommend keeping only the target and remove the project file altogether

Thanks for considering

### CODE REVIEW

File: SmartcarAuth/UIGenerator/SmartcarAuthPickerGenerator.swift [hunk 1]
```diff
@@ -114,9 +135,19 @@ public class SmartcarAuthPickerGenerator: SmartcarAuthUIGenerator, UIPickerViewD
         Hides the picker, invisButton, and toolBar
     */
     @objc func hidePickerView() {
-        picker.isHidden = true
-        invisButton.isHidden = true
-        toolBar.isHidden = true
+        viewController.view.layoutIfNeeded()
+        
+        //Animate disappearance
+        UIView.animate(withDuration: 0.3, animations: {
```
Comment (zz60) [inline SmartcarAuth/UIGenerator/SmartcarAuthPickerGenerator.swift]: I think this is causing `testHidePickerButtonPress()` to fail due to the delay. We have turned on automated test for PRs. Can you change up the test and make another commit?

**Comment by dvkch** [comment]
@zz60 I have updated the tests to take animation and new hiding technique into account

**Comment by zz60** [comment]
Looks good! Thank you for the changes!
